### PR TITLE
Jupyter notebook sample + update readme

### PR DESCRIPTION
**Pull request by dsargent**
First cut -- I tried to make a generic, domain-agnostic notebook with "fill-in-the-blanks" parts for users to create their own apps using RIO. I based it on @ofrancon's XDE notebook `2.2.Random_forest_regressor_predictor_training` and tried to remove anything domain specific (XDE related).

@VsonicV Jupyter notebooks are unreviewable in Github. The only way is to fire up `jupyter notebook` locally and review it in the browser. I can help if you're not familiar.

I haven't tested this "live" so there may be typos and bugs in the notebook. But at least it's a start.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Copyright (C) 2019 Cognizant Digital Business, Cognizant Artifical Intelligence Practice.  All rights reserved.
 
+# For gRPC
 gpflow==1.3.0
 grpcio-tools==1.16.0
 grpcio==1.16.0
 protobuf==3.9.1
+
+# For jupyter notebooks
+jupyter==1.0.0
```
Comment (ofrancon) [inline requirements.txt]: That's probably not the library you want to include. For XDE we use `notebook==5.7.8`, and there's even a version `6.x.x` now.

Comment (ofrancon) [reply requirements.txt]: BTW I'm not sure we want a dependency on jupyter notebooks in `requirements.txt`. It's totally fine to use `rio-sdk` outside of jupyter notebooks.
I guess it's fine to have it in the requirements here, though it's not in `setup.py`. I don't really like the mismatch, but I guess it's the only way to get the sample notebook to work out of the box.

File: notebooks/sample_rio_notebook.ipynb [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Improving predictions using RIO service\n",
+    "\n",
+    "This example guides you through connecting to the RIO service, and improving the predictions from your existing model (for example, your trained neural network) as well as computing the improvement of RIO over your model's un-enhanced predictions.\n",
+    "\n",
+    "To run this sample, you will need:\n",
+    "\n",
+    "1) A trained model of some kind (e.g. a neural net) that can generate predictions from test data\n",
```
Comment (ofrancon) [inline notebooks/sample_rio_notebook.ipynb]: a RandomForest regressor would much simpler for an example than a neural net.

File: notebooks/sample_rio_notebook.ipynb [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def train_my_model():\n",
+    "    # TODO: implement this function\n",
```
Comment (ofrancon) [inline notebooks/sample_rio_notebook.ipynb]: We could actually give a small pretrained-model here.

File: notebooks/sample_rio_notebook.ipynb [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+    "from rio_sdk.rio_service import RioService"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "RIO_HOST = \"rio.example.com\" # replace with hostname for your RIO service\n",
```
Comment (ofrancon) [inline notebooks/sample_rio_notebook.ipynb]: Good, not naming an actual service :)

**Comment by ofrancon** [comment]
In case it's more practical to keep everything in a single repo (I guess it is), I'd:
- add scikit-learn to `requirements.txt` (but NOT to `setup.py`)
- start the notebook by downloading a data set from UCI
- train a model, then a RIO model and compare their errors
- and ignore checkpoints in `.gitignore:
```
# ipython notebook checkpoints
.ipynb_checkpoints/
```

**Comment by dsargent** [comment]
@ofrancon is this PR moot now that there is a [sample notebook](https://github.com/leaf-ai/rio-service/blob/master/rio_train_save_load_predict.ipynb) in the `rio-service` project?

**Comment by ofrancon** [comment]
I think it is, yes - I think the rio-service notebook does the job. It actually replaces `rio-sdk` in a way now.

**Pull request closed without merging**